### ISSUE TITLE
Fold this repo into ACLPUB?

### ISSUE THREAD

**Issue by mjpost** [acl-org/acl-pub#23]
This repo is confusing to me. The top-level documentation is poor and it seems to contain scattered bits and pieces of documentation along with important style files. I also hate that the main branch is gh-pages. I believe this repo is a descendant of one Colin and I created for NAACL in 2013 to consolidate information about the pubchair process.

I suggest that we extract the style files and copyright form into ACLPUB so that we have a single place for publications-related materials. Higher-level pub chair instructions are better placed on the ACL Wiki or some other place, I think.

Are there any objections or things I'm missing?

@acl-org/acl-style @davidweichiang @danielgildea

**Comment by davidweichiang** [comment]
Is this consistent with the discussion in #13?

**Comment by davidweichiang** [comment]
Also, maybe these repos should be removed:
- https://github.com/acl-org/START
- https://github.com/acl-org/archive
- https://github.com/aclpub/aclpub
- https://github.com/naacl-org/aclpub

**Comment by mjpost** [comment]
Yes—what do you think about making issues on each of those repos suggesting they be deleted? Or maybe we could just ask @desilinguist to delete them?

I see that it was me at least in part who prevented a merge. One way we could ensure that things proceed smoothly would be to adopt a master branch protection system that requires a review to merge.

**Comment by davidweichiang** [comment]
I kind of suspect that no one would mind if @desilinguist just deleted them.....

If I understand you correctly, you want conference pub chairs to work only on branches, and there would be a small group of maintainers, not including pub chairs, that would approve changes to the master branch. I think that sounds okay; I think it might be simpler still to let conference pub chairs make forks of the main repository. In either case, we'd be depending on pub chairs to make PRs for the improvements they make, but it would be easy for pub chairs to forget to do that.

**Comment by mjpost** [comment]
Yeah, I agree that forks are fine, and actually maybe preferable to branches. My main goals are to preserve the sanctity of the official repo—both protecting its contents, and also making clear its official status.

I agree it will probably be on us / the maintainers to pull in any changes that occur. I also suspect that changes may occur outside the repo, for example with Overleaf becoming the new de facto place for changes to style files.

**Comment by davidweichiang** [comment]
#17

**Comment by mjpost** [comment]
Renaming in light of conversation that revealed the core issue.

**Comment by mjpost** [comment]
Nitin just removed the first two from [David's comment above](https://github.com/acl-org/acl-pub/issues/23#issuecomment-559834009).

**Comment by mjpost** [comment]
And I removed https://github.com/naacl-org/aclpub.

I'm not sure who the owner of https://github.com/aclpub/aclpub is.

**Comment by alexanderkoller** [comment]
> I'm not sure who the owner of https://github.com/aclpub/aclpub is.

Me! Yusuke Miyao and I set it up when we were publication chairs for ACL 2014 and then tried to pass it on to future pub chairs, but I think the chain got broken at some point.

The sole purpose of that repository is to host this website:  https://aclpub.github.io/aclpub/

Do you want to have a look if there's anything on that site that is worth integrating into this repository? Otherwise I'd be happy to archive it.

**Comment by mjpost** [comment]
Ah, that’s actually much prettier than the official one:

	https://acl-org.github.io/acl-pub/ <https://acl-org.github.io/acl-pub/>

I’d like to redesign the official one based on what you have here. Is that okay with you? Then we can talk about archiving yours.

matt

**Comment by alexanderkoller** [comment]
Upon rereading our notes from 2014, I am actually quite impressed with the degree to which we documented detailed knowledge that each publication chair otherwise needs to figure out for themselves.

A lot of the technical details are probably outdated with more modern ACLPUB, but it could be nice if you'd like to pass on the details on where to draw the line with the formatting rules and the notes on how to interact with the other chairs and the book chairs.

**Comment by alexanderkoller** [comment]
PS I really appreciate your efforts in both providing infrastructure for future publication chairs and in getting sister repos cleaned up.

**Comment by mjpost** [comment]
Yes, it seems there is good information in there. I will merge the two repos (aclpub and acl-pub) into pubdocs in the near future and report back here when I've made progress.

**Comment by mjpost** [comment]
@alexanderkoller, if you're happy enough with [the new merged site](https://acl-org.github.io/ACLPUB/) (with the understanding that further changes are coming), would you mind deleting or at least archiving https://github.com/aclpub/aclpub?

**Comment by alexanderkoller** [comment]
Hi guys, I just archived aclpub/aclpub and set its visibility to private.

Thank you for all your efforts in consolidating the documentation for future publication chairs. I know how much work it takes.
